src/dynamodb.js

```diff
@@ -52,8 +84,25 @@ async function destroyTables (dynamoClient, uniqueIdentifier) {
     tablesToDestroy
       .map(async TableName => {
         try {
-          await dynamoClient.deleteTable({ TableName }).promise();
-          await dynamoClient.waitFor('tableNotExists', { TableName }).promise();
+          const deleteRequest = dynamoClient.deleteTable({ TableName });
+
+          deleteRequest.on('retry', response => {
```

Comment:
I am getting errors now on Jenkins where the resource isn't deleted.  I believe they are retryable, but this is causing them not to be retried.

Reply:
Oh no. What errors are you seeing that should be retried? We should be able to add some logic in to fix that.

Reply:
Looks like you started the PR already: https://github.com/lifeomic/lambda-tools/pull/166/files/4462c4faa91ec431a5d7a93e82e9f888f4936462..9166b2d7c3bf42fa305d015549da135259deceb8